fearless/Modules/NewWallet/WalletSend/WalletSendInteractor.swift

```diff
@@ -55,12 +55,16 @@ final class WalletSendInteractor: RuntimeConstantFetching {
             self?.presenter?.didReceiveBlockDuration(result: result)
         }
 
-        fetchConstant(
-            for: .existentialDeposit,
-            runtimeCodingService: runtimeService,
-            operationManager: operationManager
-        ) { [weak self] (result: Result<BigUInt, Error>) in
-            self?.presenter?.didReceiveMinimumBalance(result: result)
+        if chain.isOrml {
```

Comment:
I already found same logic in several modules, mb we can use some common handler for all of them.